compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -112,35 +112,35 @@ bool get_channel_dim_index(CircleConst *node, loco::TensorShape &dimension, int
     auto tw_conv = dynamic_cast<CircleTransposeConv *>(out);
     auto fc = dynamic_cast<CircleFullyConnected *>(out);
 
-    if (conv != nullptr && conv->filter() == node)
+    if ((conv != nullptr && conv->filter() == node) ||
+        (tw_conv != nullptr && tw_conv->filter() == node)) // OHWI
     {
       assert(node->rank() == 4);
       dimension.dim(0).set(node->dim(0).value());
       dimension.dim(1).set(node->dim(1).value());
       dimension.dim(2).set(node->dim(2).value());
       dimension.dim(3).set(node->dim(3).value());
-      channel_dim_index = 3;
+      channel_dim_index = 0; // I set channel_dim_index based on "O"
```

Comment:
Q) from `I set channel_dim_index ...`; is `I` from Input or something else?

Reply:
In the case of ```depthwise conv(tw_conv)```, the kernel of 3 dimensions is quantized by the number of channel multiplications.
```number of channel multiplications``` seems ```output_channel```.